src/test/resources/features/Roles.feature

```diff
@@ -6,22 +6,31 @@ Feature: Role features
     And an authorized client
 
   Scenario: Authorized client creates Role
-    When the authorized client sends a "POST" request
+    Given that the authorized client intends to add a new Role
```

Comment:
Not sure about the "intends" — is it necessary? "Given that the client is authorized | When the client sends a request with the kv pairs…